sechub-solution/helm/sechub-server/templates/deployment.yaml

```
@@ -152,6 +172,9 @@ spec:
         # Base url for sechub server - e.g. https://sechub.example.com
         - name: SECHUB_SERVER_BASEURL
           value: "{{ .Values.server.baseURL.userAccess }}"
+        # Static resources for login page
+        - name: SPRING_WEB_RESOURCES_STATICLOCATIONS
+          value: "{{ .Values.staticResources.spring.staticLocations }}"
```

The pathes are fixed so the SPRING_WEB_RESOURCES_STATICLOCATIONS can be set dependent on the value of `staticResources.enabled `
```suggestion
{{- if .Values.staticResources.enabled }}
          value: "file:/sechub/web-resources/static,classpath:/static"
{{- else }}
          value: "classpath:/static"
{{- end }}
```